blivet/devicefactory.py

```diff
@@ -336,7 +337,7 @@ def __init__(self, storage, size, disks, fstype=None, mountpoint=None,
 
         self.child_factory = None
         self.parent_factory = None
-        self.min_luks_entropy = min_luks_entropy
+        luks_data.min_entropy = min_luks_entropy
```

Comment:
And maybe store the value for future devices if it is not set explicitly again then.